web-components/src/components/list/ListItem.test.ts

```diff
@@ -25,8 +25,8 @@ describe("ListItem", () => {
     element.disabled = false;
     await elementUpdated(element);
 
-    expect(element.tabIndex).toEqual(0);
-    expect(element.getAttribute("tabindex")).toEqual("0");
+    expect(element.tabIndex).toEqual(-1);
+    expect(element.getAttribute("tabindex")).toEqual("-1");
```

Comment:
Can you also add in List Examples disabled case?

Reply:
This test covers for the disabled case as well. added for focus.